dockers/docker-fpm-frr/start.sh

```diff
@@ -33,6 +33,13 @@ rm -f /var/run/rsyslogd.pid
 
 supervisorctl start rsyslogd
 
+# start eoiu pulling, only if configured so
+if [[ $(sonic-cfggen -d -v 'WARM_RESTART.bgp.bgp_eoiu') == 'true' ]]; then
+		supervisorctl start bgp_eoiu_marker
+fi
+
+supervisorctl start bgpcfgd
```

Comment:
Looks like this is added accidently? It was moved to end as part of  https://github.com/Azure/sonic-buildimage/pull/3627

Reply:
https://github.com/Azure/sonic-buildimage/pull/3752